src/commands/list-wcw-topics.ts

```diff
@@ -5,19 +5,19 @@ import { listTopics } from '../lib/listTopics';
 import { formatTopics } from '../lib/formatTopics';
 
 export const data = new SlashCommandBuilder()
-    .setName('list-topics')
+    .setName('list-wcw-topics')
     .setDescription('List topics for 💦 Water Cooler Wednesdays');
 
 export async function execute(interaction: CommandInteraction<CacheType>) {
     const member = interaction.member as GuildMember;
     const topics = await listTopics();
 
-    const notUsed = topics.filter(item => !item.used);
-    const used = topics.filter(item => item.used);
+    const notUsed = topics.filter((item: { used: boolean; }) => !item.used);
+    const used = topics.filter((item: { used: boolean; }) => item.used);
```

Comment:
`item` is already typed _(photo below)_ so why is there an explicit type here. I assume this was when we were going through the prisma issues but since that's resolved this isn't needed.
![CleanShot 2023-01-25 at 21 59 13@2x](https://user-images.githubusercontent.com/17864504/214755858-ebebbde7-3931-4111-9b4d-0b0dc00ddf4b.png)

```suggestion
    const notUsed = topics.filter(item => !item.used);
    const used = topics.filter(item => item.used);
```

Reply:
Yeah, at some point, it was complaining that item property wasn't recognised, so I added that to make the warning go away. Seems happy now though, so will revert